integration_test/gce/gce_testing.go

```diff
@@ -1262,15 +1264,15 @@ func attemptCreateInstance(ctx context.Context, logger *log.Logger, options VMOp
 		vm.OS = *os
 	}
 
-	if vm.OS.ID == "opensuse" || vm.OS.ID == "opensuse-leap" || vm.OS.ID == "sles" || vm.OS.ID == "sles_sap" {
+	if IsSUSE(vm.ImageSpec) {
 		// Set download.max_silent_tries to 5 (by default, it is commented out in
 		// the config file). This should help with issues like b/211003972.
 		if _, err := RunRemotely(ctx, logger, vm, "sudo sed -i -E 's/.*download.max_silent_tries.*/download.max_silent_tries = 5/g' /etc/zypp/zypp.conf"); err != nil {
 			return nil, fmt.Errorf("attemptCreateInstance() failed to configure retries in zypp.conf: %v", err)
 		}
 	}
 
-	if strings.HasPrefix(vm.ImageSpec, "sles-") {
+	if vm.OS.ID == "sles" || vm.OS.ID != "sles_sap" {
```

Comment:
This seems wrong, won't this almost always be true?

Reply:
Yes, that's wrong i made a typo when setting this `vm.OS.ID != "sles_sap"`. Should be `vm.OS.ID == "sles_sap"`, since i'm trying to do an `IsSLES` function.